### DISCUSSION TITLE
I get server error 500

### DISCUSSION THREAD

**Discussion by mihaeldonko** [microweber/microweber#912] [Q&A]
I have this issue and dont know what to do.
![image](https://user-images.githubusercontent.com/109346556/182387439-adacbb1b-3d8d-4f10-b9c7-540d4ed705e7.png)

**Comment by peter-mw** [comment]
Hello, 

you can enable debug error by editing the `config/app.php` file and setting  `'debug' => true`

**Comment by engr-sakil** [comment]
I am also face this problem when i try to install MW in local server.I solved this issue now share with you.I hope it will be work.

Open php.ini file and remove semicolon from ;extension=intl 

extension=intl 

and restart the Apache server

**Comment by mihaeldonko** [comment]
Thank you @SAKIL2037 and @peter-mw for anwsers. I fixed this issue by installing one version older microweber from sourceforge this fixed my issue of running microweber in local server.

**Comment by engr-sakil** [comment]
@mihaeldonko  If you follow my provided way. You can use the latest version in your local server.
If you need any another help you can contact with me directly any time through my linkedin. 
Linkedin Profile: [https://www.linkedin.com/in/sakilmia/]

Regards
Sakil Mia
Software Engineer
